### PR TITLE
DFIR IRIS Integration contribution

### PR DESCRIPTION
**Pull request by Enigmatyk**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
New integration

## Description
IRIS is a collaborative platform aiming to help incident responders to share technical details during investigations

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

File: Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+    case_id = args.get("case_id")
+    results_str = dfir_iris.close_case(case_id)
+
+    readable_output = tableToMarkdown('Command successfully sent to IRIS DFIR"', results_str, removeNull=True)
+
+    return CommandResults(
+        outputs_prefix='IRIS',
+        outputs_key_field='',
+        readable_output=readable_output,
+        outputs=results_str,
```
[the remark this answers is not in the document]
Comment (yucohen) [reply Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py:301]: if it just retrieves an str no need for outputs in this case.

**Review by yucohen — CHANGES_REQUESTED** [review]
Thanks for your contribution! Great job!
1. Please see my review comments.
2. make sure to fix all build steps.
3. Isn't mandatory, but adding a detailed description on how to get api tokens to `IRISDFIR_description.md` file would be really useful to the customer using this integration.

Dont hesitate to reach out if you have any questions here or on slack DFIR :)

File: Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+        response = requests.post(f'{self.api_endpoint}/case/notes/groups/add', headers=self.headers,
+                                 verify=verify_cert, proxies=proxies, json=body
+                                 )
+
+        if response.status_code == 200:
+            cases = response.json()
+            if cases:
+                return cases['data']
+            else:
+                return cases['message']
```
[the remark this answers is not in the document]
Comment (Enigmatyk) [reply Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py:138]: Removed.

File: Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py [hunk 3]
```diff
@@ -0,0 +1,468 @@
...
+        response = requests.post(f'{self.api_endpoint}/case/notes/add', headers=self.headers,
+                                 verify=verify_cert, proxies=proxies, json=body
+                                 )
+
+        if response.status_code == 200:
+            cases = response.json()
+            if cases:
+                return cases['data']
+            else:
+                return cases['message']
```
[the remark this answers is not in the document]
Comment (Enigmatyk) [reply Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py:160]: Removed.

File: Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py [hunk 4]
```diff
@@ -0,0 +1,468 @@
...
+        response = requests.get(f'{self.api_endpoint}/case/notes/groups/list?cid={case_id}', headers=self.headers,
+                                verify=verify_cert, proxies=proxies
+                                )
+
+        if response.status_code == 200:
+            cases = response.json()
+            if cases:
+                return cases['data']
+            else:
+                return cases['message']
```
[the remark this answers is not in the document]
Comment (Enigmatyk) [reply Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py:175]: Removed.

File: Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py [hunk 5]
```diff
@@ -0,0 +1,468 @@
...
+        response = requests.get(f'{self.api_endpoint}/case/ioc/list?cid={case_id}', headers=self.headers,
+                                verify=verify_cert, proxies=proxies
+                                )
+
+        if response.status_code == 200:
+            cases = response.json()
+            if cases:
+                return cases['data']
+            else:
+                return cases['message']
```
[the remark this answers is not in the document]
Comment (Enigmatyk) [reply Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py:190]: Removed.

File: Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Enigmatyk) [reply Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.py:301]: change the name to results to not create confusion.

**Comment by yucohen** [comment]
@Enigmatyk Great! approving!
waiting for your confirmation before merging :)

**Review by yucohen — APPROVED** [review]

**Review by yucohen — APPROVED** [review]
Great!!

**Pull request merged**